### PR TITLE
Create the calculator UI design

### PR DESCRIPTION
**Pull request by Ibnballo1**
In this PR, I did the following:

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, I created a new Calculator.js file.
- In Calculator.js, I created a React component that matches the design given to us

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <Calculator />
+  </div>
+);
```
Comment (gemmen29) [inline src/App.js:8]: - [ ] Kindly convert the app component to class component as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Ibnballo1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Ibnballo1) [reply src/App.js:8]: Thanks, @gemmen29, but I assume we are only required to use the class component for Calculator.js?

Comment (Ibnballo1) [reply src/App.js:8]: @Fixed

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Ibnballo1👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**